### PR TITLE
Migrate kubernetes secretupdater chart from internal helm repo to github

### PR DESCRIPTION
**Pull request by egeland**

### CODE REVIEW

File: stable/k8s-secret-updater/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes SecretUpdater
+name: secret-updater
+version: 0.1.1
+maintainers:
+  - name: FFX Blue Operations
```
Comment (matthope) [inline stable/k8s-secret-updater/Chart.yaml]: Please update/correct

File: stable/k8s-secret-updater/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Kubernetes Secret Updater
+
+This is used in conjunction with Confidant, to update the secrets in k8s namespaces.
```
Comment (matthope) [inline stable/k8s-secret-updater/README.md]: Please add URL to confidant doco

File: stable/k8s-secret-updater/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    email: blueops@fairfaxmedia.com.au
```
Comment (matthope) [inline stable/k8s-secret-updater/Chart.yaml]: please update/correct this email address

File: stable/k8s-secret-updater/README.md (hunk 2 above)
Comment (matthope) [inline stable/k8s-secret-updater/README.md]: Please add a link to https://github.com/fairfaxmedia/k8s-secret-updater

Comment (matthope) [reply stable/k8s-secret-updater/README.md]: https://github.com/lyft/confidant

**Review by matthope — CHANGES_REQUESTED** [review]

File: stable/k8s-secret-updater/README.md (hunk 2 above)
Comment (matthope) [reply stable/k8s-secret-updater/README.md]: ... and https://hub.docker.com/r/fairfaxmedia/k8s-secret-updater - there's some good doco there :)

File: stable/k8s-secret-updater/Chart.yaml (hunk 3 above)
Comment (egeland) [reply stable/k8s-secret-updater/Chart.yaml]: that's the same as the other charts in this repo...

File: stable/k8s-secret-updater/README.md (hunk 2 above)
Comment (egeland) [reply stable/k8s-secret-updater/README.md]: Added!

Comment (egeland) [reply stable/k8s-secret-updater/README.md]: Added!

File: stable/k8s-secret-updater/Chart.yaml (hunk 3 above)
Comment (egeland) [reply stable/k8s-secret-updater/Chart.yaml]: did we get github@fairfaxmedia.com.au set up?

Comment (matthope) [reply stable/k8s-secret-updater/Chart.yaml]: <private conversation follows>

Comment (egeland) [reply stable/k8s-secret-updater/Chart.yaml]: Edited!

**Review by matthope — APPROVED** [review]

**Pull request merged**